packages/dds/tree/src/test/shared-tree/fuzz/fuzzUtils.ts

```diff
@@ -24,24 +24,59 @@ import {
 	forEachNodeInSubtree,
 	moveToDetachedField,
 } from "../../../core/index.js";
-import { SchemaBuilder, leaf, typedJsonCursor } from "../../../domains/index.js";
-import {
-	Any,
-	FieldKinds,
-	FlexFieldSchema,
-	type FlexTreeObjectNodeTyped,
-	type LeafNodeSchema,
-	type SchemaLibrary,
-	intoStoredSchema,
-} from "../../../feature-libraries/index.js";
-import type { ITreeCheckout, SharedTree, TreeContent } from "../../../shared-tree/index.js";
+import type { ITreeCheckout, SharedTree } from "../../../shared-tree/index.js";
 import { testSrcPath } from "../../testSrcPath.cjs";
 import { expectEqualPaths } from "../../utils.js";
-
-const builder = new SchemaBuilder({ scope: "treefuzz", libraries: [leaf.library] });
+import type {
+	NodeBuilderData,
+	// eslint-disable-next-line import/no-internal-modules
+} from "../../../simple-tree/schemaTypes.js";
+import {
+	SchemaFactory,
+	TreeViewConfiguration,
+	type TreeNodeSchema,
+	type ValidateRecursiveSchema,
+} from "../../../simple-tree/index.js";
+
+const builder = new SchemaFactory("treeFuzz");
+export class GUIDNode extends builder.object("GuidNode", {
+	value: builder.optional(builder.string),
+}) {}
+
+export class FuzzStringNode extends builder.object("FuzzStringNode", {
+	stringValue: builder.required(builder.string),
+}) {}
+export class FuzzNumberNode extends builder.object("FuzzNumberNode", {
+	value: builder.required(builder.number),
+}) {}
+export class FuzzHandleNode extends builder.object("FuzzHandleNode", {
+	value: builder.required(builder.handle),
+}) {}
+
+export type InitialAllowedFuzzTypes =
+	| FuzzStringNode
+	| FuzzNumberNode
+	| FuzzHandleNode
+	| GUIDNode
+	| FuzzNode;
+
+const initialAllowedTypes = [
+	FuzzStringNode,
+	FuzzNumberNode,
+	FuzzHandleNode,
+	GUIDNode,
+	() => FuzzNode,
+] as const;
+
+export class SequenceChildren extends builder.arrayRecursive(
```

Comment:
Should this be "ArrayChildren" or something to avoid a reference to an implementation detail (sequences)?

Reply:
Updated `SequenceChildren` to `ArrayChildren`